plugins/modules/instance.py

```diff
@@ -83,6 +83,13 @@
         type=FieldType.string,
         description=["The filesystem to create this disk with."],
     ),
+    "disk_encryption": SpecField(
+        type=FieldType.string,
+        description="The disk encryption status of this disk."
+        + "NOTE: Disk encryption may not currently be available to all users.",
+        choices=["enabled", "disabled"],
+        default="enabled",
```

Comment:
It might make sense to defer the default logic to the API since there could be a case where a user wants to update certain fields of an LDE-disabled instance without implicitly trying to update `disk_encryption`, which raises an error 👀

Reply:
Good call! Fixed.